mne/io/edf/edf.py

```diff
@@ -1614,6 +1651,10 @@ def read_raw_edf(
     %(preload)s
     %(units_edf_bdf_io)s
     %(encoding_edf)s
+    exclude_after_unique : bool
+        If True, exclude channels are searched for after they have been made
+        unique. This is useful to choose channels that have been made unique
+        by adding a suffix.
```

Comment:
DRY (don't repeat yourself) -- this should live in an entry in `mne/utils/docs.py` and be used all three places